cmd/goexports/goexports.go

```diff
@@ -61,16 +66,21 @@ func init() {
 				"{{$key}}": reflect.ValueOf({{$value.Name}}),
 			{{end -}}
 		{{end}}
+		{{- end}}
 
+		{{- if len .Typ | ne 0}}
```

Comment:
This check seems not needed because the range syntax already support empty map.

Reply:
I would place these conditions (`.Typ` and `.Wrap`) one line up. At the moment they still render the blank line if the map is empty, leaving potentially 2 blank lines in the case where there is no type or interface symbol to write.

Reply:
to handle this case, one would need to blank-import the package being wrapped.
(I'll add that. I stumbled on this for the _e.g._ `runtime/race` package.)